drift-transport-netty/src/main/java/com/facebook/drift/transport/netty/server/DriftNettyServerConfig.java

```diff
@@ -57,6 +58,7 @@
     private Duration sessionTimeout = new Duration(1, DAYS);
 
     private boolean assumeClientsSupportOutOfOrderResponses = true;
+    private boolean nativeTransportEnabled = true;
```

Comment:
Same thing here. I would recommend setting it to `false` by default.

Reply:
done